src/vcpkg/dependencies.cpp

```diff
@@ -1250,36 +1250,41 @@ namespace vcpkg::Dependencies
                 std::vector<std::string> features;
             };
 
-            // This object contains the current version within a given column.
-            // Each "string" scheme text is treated as a separate column
-            // "relaxed" versions all share the same column
+            // This object contains the current version within a given version sheme (except for the "string" scheme,
+            // there we save an object for every version)
             struct VersionSchemeInfo
             {
                 Versions::Scheme scheme;
                 const SourceControlFileLocation* scfl = nullptr;
                 Versions::Version version;
                 // This tracks a list of constraint sources for debugging purposes
                 std::vector<std::string> origins;
+                // mapping from feature name -> dependencies of this feature
                 std::map<std::string, std::vector<FeatureSpec>> deps;
 
                 bool is_less_than(const Versions::Version& new_ver) const;
             };
 
             struct PackageNode
             {
+                // Mapping from version to the newest version in the cooresponding version sheme
```

Comment:
```suggestion
                // Mapping from version to the newest version in the corresponding version scheme
                // For example, given the versions:
                //   - "version-string": "1.0.0"
                //   - "version": "1.0.1"
                //   - "version": "1.0.2"
                // you'd have a map:
                //   {
                //     "1.0.0": { "version-string": "1.0.0" },
                //     "1.0.1": { "version": "1.0.2" },
                //     "1.0.2": { "version": "1.0.2" }
                //  }
```

(assuming my understanding of this is correct)